src/main/java/com/liveramp/captain/daemon/CaptainJoblet.java

```diff
@@ -188,6 +189,10 @@ private void submitRequest(CaptainRequestConfig config) throws DaemonException {
       notifier.notify(subject, e, CaptainNotifier.NotificationLevel.ERROR);
       requestUpdater.cancel(id);
       return;
+    } catch (CaptainTransientFailureException ce) {
+      String subject = String.format("%s: Transient failure while submitting requests %s. Do nothing", CaptainAlertHelpers.getHostName(), id);
```

Comment:
(This one says do nothing, the others say "Doing nothing")

Reply:
let me update to be consistent